osu.Game/Screens/Select/FooterButton.cs

```diff
@@ -74,11 +78,23 @@ public FooterButton()
                     Colour = Color4.White,
                     Alpha = 0,
                 },
-                light = new Box
+                LightContainer = new Container
                 {
-                    Height = 4,
-                    EdgeSmoothness = new Vector2(2, 0),
-                    RelativeSizeAxes = Axes.X,
+                    RelativeSizeAxes = Axes.Both,
+                    //Make container for light slightly smaller than parent.
+                    Size = new Vector2(0.996f, 4),
+                    //Anchor is at top right because ModIcons are added to the right
+                    //and rounding errors (relative size od x) are hidden to the left.
+                    Anchor = Anchor.TopRight,
+                    Origin = Anchor.TopRight,
+                    Child = light = new Box
+                    {
+                        Size = new Vector2(0.998f, 4),
```

Comment:
Can inherit parent's sizings (width and height are 1 by default):
```suggestion
                        Width = 0.998f,
                        RelativeSizeAxes = Axes.Both,
```

Reply:
I think I'm missing something here. It does not inherit.

This is the result:
![osu_2019-09-27_21-11-55](https://user-images.githubusercontent.com/43511642/65771818-9ccde400-e16b-11e9-87e0-9ff0debf279c.png)

Reply:
Actually, to be more correct here, the container should set `AutoSizeAxes` to `Y` and `RelativeSizeAxes` to `X` and move the height to the light box. (so it would auto-size height of the container to the light box which would be `4`).